src/components/pages/AddChild/RenderAddChild.js

```diff
@@ -12,7 +9,6 @@ const { Title } = Typography;
 const RenderAddChild = props => {
   return (
     <Layout className="add-child">
```

Comment:
All of the following changes are removing the parentNavTopBar as it no longer needs to be called in the individual components.